src/main/java/edu/wpi/first/pathweaver/ProjectPreferences.java

```diff
@@ -111,6 +112,7 @@ public Values(double timeStep, double maxVelocity, double maxAcceleration, doubl
       this.maxAcceleration = maxAcceleration;
       this.maxJerk = maxJerk;
       this.wheelBase = wheelBase;
+      this.game  = "POWERUP";
```

Comment:
This should be a constructor parameter for a value class. You could make a static factory method for a game-specific value object with the game pre-set, but hardcoding it in the constructor seems redundant.